controller/user.go

```diff
@@ -0,0 +1,57 @@
+package controller
+
+import (
+	"fmt"
+	. "WeKnow_api/helper"
+	. "WeKnow_api/pgModel"
+	"encoding/json"
+	"net/http"
+)
+
+var db = Connect()
```

Comment:
@mikey2020 I will rather we don't use a period before imported packages. This will not only pollute the namespace but will also impact the readability of the code.
I knew where `Connect` was coming from only because I'm still familiar with everything in the codebase, this might change in the future. I think having something like `pgModel.Connect()` would be fair. Also for the helper; if you are not comfortable with `helper` being continually repeated, you can provide an alternate qualifier for it in the namespace; something like
```
import h "WeKnow_api/helper"
```
then subsequently, you can now use `h.RespondWithError`